Blocks/URL/url.h

```diff
@@ -515,6 +515,23 @@ struct URLPathArgs {
 
   void add(const std::string& arg) { args_.push_back(arg); }
 
+  std::string ComposeURLPathFromArgs() const {
+    std::string result;
+    for (auto cit = args_.crbegin(); cit != args_.crend(); ++cit) {
```

Comment:
Won't `for (const auto& component : (*this)) {` do the trick here? It's already a reverse iterator.

Reply:
Yes, it will